packages/cli/src/lib/graphql/admin/link-storefront.ts

```diff
@@ -0,0 +1,21 @@
+import {gql} from 'graphql-request';
+
+export const LinkStorefrontQuery = gql`
```

Comment:
Is the `gql` tag required to make a request here or can we just go with strings?

```
`#graphql
  query LinkStorefront {
    ...
  }
`
```

Reply:
I'll give this a go, would be great to avoid the import.